### PR TITLE
add mint_nft script

### PR DESCRIPTION
**Pull request by kozayupapa**
mint nft to solana on chain script.

minted NFT example:
 https://explorer.solana.com/address/4NTVmi1uu951XCEUkmqiM7Gc3kBVU1JQavJHTN4xPxnB?cluster=devnet

featurepoint nft token sample 1: https://explorer.solana.com/address/8A5UtHiWpa87kXgbPwznhJ8zWKzdnQuzeGcjE8yTp4fS?cluster=devnet
sample2 :
https://explorer.solana.com/address/A7NmWjRUsAjSM1RQ9X1UQd183VtJPu6j1moFASv4Dji7?cluster=devnet

### CODE REVIEW

File: Sandbox/Solana/src/mint_nft.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import { Connection, Keypair, PublicKey } from "@solana/web3.js";
+import { Metaplex, keypairIdentity, bundlrStorage, toMetaplexFile, toBigNumber } from "@metaplex-foundation/js";
+import * as fs from 'fs';
+import secret = require('/Users/takehararyuuji/ryu-solana-wallet/my-keypair.json');
```
Comment (Nasfame) [inline Sandbox/Solana/src/mint_nft.ts:4]: Can u store the path as environment variable? Also we are more or like going to deploy on Ubuntu so let's remove windows paths and use relative paths...

**Review by Nasfame — CHANGES_REQUESTED** [review]

File: Sandbox/Solana/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
```
Comment (Nasfame) [inline Sandbox/Solana/tsconfig.json:3]: Use es6 modules...

File: Sandbox/Solana/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    "declaration": true,
+    "removeComments": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "allowSyntheticDefaultImports": true,
+    "resolveJsonModule": true,
+    "target": "es2017",
```
Comment (Nasfame) [inline Sandbox/Solana/tsconfig.json:10]: Use es2022

File: Sandbox/Solana/yarn.lock [hunk 4]
```diff
@@ -0,0 +1,3086 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (Nasfame) [inline Sandbox/Solana/yarn.lock:1]: Remove lock files. Add it to gitignore

File: Sandbox/Solana/tsconfig.json (hunk 2 above)
Comment (kozayupapa) [reply Sandbox/Solana/tsconfig.json:3]: applied

File: Sandbox/Solana/tsconfig.json (hunk 3 above)
Comment (kozayupapa) [reply Sandbox/Solana/tsconfig.json:10]: applied

File: Sandbox/Solana/yarn.lock (hunk 4 above)
Comment (kozayupapa) [reply Sandbox/Solana/yarn.lock:1]: I think we should include lock file to github to fix versions of each module.
why do you think remove lock file?

Comment (Nasfame) [reply Sandbox/Solana/yarn.lock:1]: package.json locks the versions

Comment (Nasfame) [reply Sandbox/Solana/yarn.lock:1]: lock files create git log noise. Its better to have it removed

Comment (Nasfame) [reply Sandbox/Solana/yarn.lock:1]: if there a transitive deps conflicts, then okay you can use yarn

**Review by Nasfame — APPROVED** [review]
LGTM

**Pull request merged**